src/components/common/Input/index.js

```diff
@@ -18,6 +19,8 @@ const Input = ({ label, value, onChangeText, error, active, touched, ...props })
           style={[styles.input, touched && !!error && styles.inputError]}
           value={value}
           onChangeText={onChangeText}
+          placeholderTextColor={GRAY}
+          placeholder={placeholder.toUpperCase()}
```

Comment:
FYI: I tried to work with textTransform but it didn't work. For some reason it doesn't apply the style correctly to TextInput.